doc/tutorial/01_introduction/introduction_dna4.cpp

```diff
@@ -0,0 +1,23 @@
+//! [debug]
+#include <iostream> // for std::cerr
+#include <vector>   // for std::vector
+
+#include <seqan3/alphabet/all.hpp>             // for all alphabet related stuff
+#include <seqan3/alphabet/nucleotide/dna4.hpp> // for only dna4
+#include <seqan3/core/debug_stream.hpp>        // for debug_stream
+
+int main()
+{
+    std::vector<seqan3::dna4> vec{seqan3::assign_char_to('A', seqan3::dna4{}),
+                                  seqan3::assign_char_to('c', seqan3::dna4{}),
+                                  seqan3::assign_char_to('G', seqan3::dna4{}),
+                                  seqan3::assign_char_to('T', seqan3::dna4{})};
+    // the above is a little cumbersome because we don\t allow implicit conversions between our alphabets and `char`
+    // there is a more convenient way:
```

Comment:
```suggestion
    // The above is a little cumbersome because we don't allow implicit conversions between our alphabets and `char`.
    // There is a more convenient way:
```